resources/views/competitions/create.blade.php

```diff
@@ -4,4 +4,34 @@
 
     <h1>Create A New Competition</h1>
 
+    <div class='container__block'>
+        <div class='wrapper'>
+            {!! Form::open(['route' => 'competitions.store']) !!}
+                <div class="form-item">
+                    {{ Form::label('campaign_id', 'Campaign ID:', ['class' => 'field-label']) }}
+                    {{ Form::text('campaign_id', NULL, ['class' => 'text-field']) }}
+                </div>
+
+                <div class="form-item">
+                    {{ Form::label('campaign_run_id', 'Run ID:', ['class' => 'field-label']) }}
+                    {{ Form::text('campaign_run_id', NULL, ['class' => 'text-field']) }}
+                </div>
+
+                <div class="form-item">
+                    {{ Form::label('start_date', 'Start date:', ['class' => 'field-label']) }}
+                    {{ Form::input('date', 'start_date', NULL, ['class' => 'text-field']) }}
+                </div>
+
+                <div class="form-item">
+                    {{ Form::label('end_date', 'End date:', ['class' => 'field-label']) }}
+                    {{ Form::input('date', 'end_date', NULL, ['class' => 'text-field']) }}
+                </div>
+
+                <div class="form-item">
+                    {{ Form::submit('Submit', ['class' => 'button']) }}
+                </div>
+            {!! Form::close() !!}
```

Comment:
Okie dokes, all of this would need to be re-written anyway when we decide what the actual front end will look like. This was just a quick way to get everything on the screen